### PR TITLE
feat: adds rhel 8.10 vsphere support

### PR DESCRIPTION
**Pull request by faiq**
**What problem does this PR solve?**:
Adds rhel 8.10 support for vsphere

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-NUMBER

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: images/ova/rhel-810.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+build_name: "rhel-810"
+packer_builder_type: "vsphere"
+guestinfo_datasource_slug: "https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo"
+guestinfo_datasource_ref: "v1.4.0"
+guestinfo_datasource_script: "{{guestinfo_datasource_slug}}/{{guestinfo_datasource_ref}}/install.sh"
```
Comment (supershal) [inline images/ova/rhel-810.yaml:6]:
```suggestion

```
This is not needed anymore. We have to remove it from other OS's manifest files too. I will file separate PR.

**Review by supershal — APPROVED** [review]

File: images/ova/rhel-810.yaml (hunk 1 above)
Comment (dlipovetsky) [reply images/ova/rhel-810.yaml:6]: +1. Thanks for catching this, Shalin.

**Review by dlipovetsky — APPROVED** [review]
The 8.10 tests pass. Please address @supershal 's feedback :pray:

File: images/ova/rhel-810.yaml (hunk 1 above)
Comment (faiq) [reply images/ova/rhel-810.yaml:6]: lets handle it all in a new pr cleanly please!

**Pull request merged**